internal/xray/tree_node.go

```diff
@@ -490,6 +490,10 @@ func toEmoji(gvr string) string {
 		return "🏷 "
 	case "policy/v1beta1/podsecuritypolicies":
 		return "👮‍♂️"
+	case "apiextensions.k8s.io/v1/customresourcedefinitions":
+		return "📔"
+	case "argoproj.io/v1alpha1/applications":
+		return "🏠"
```

Comment:
As a note here @derailed, since Applications can ship any type of resources, we end up with lots of unknown resource types in the tree, and 📎 is not very clear to tell them from each other (especially when they all have the same name).

I've been wondering if there could be a clean way to tell them apart:

- using a short name in brackets instead of 📎 , e.g. `[crd]`, `[cert]`, etc.
- displaying the full GVR of the resource on hover

What do you think?